README.md

```diff
@@ -9,6 +9,14 @@ Currently this tool is tied to *JSON Schema Draft 7*.
 Support for more recent JSON Schema versions are planned
 for the future.
 
+## Usage
+
+This tool can be imported as a module or run from the command line. To see usage information about the command line tool use the following command:
+
+```bash
+python -m json_schema_fuzz --help
```

Comment:
The message that comes out of this looks like
```
Usage: __main__.py [OPTIONS] SCHEMA_FILE
```
which I found confusing since I didn't and needn't type `__main__.py`.